ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -201,6 +203,10 @@ func createCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 		return fmt.Errorf("You must specify 2 comma-separated availability zones with the '--%s' flag", command.VpcAzFlag)
 	}
 
+	if validateCommaSeparatedParam(cfnParams, cloudformation.ParameterKeyInstanceRole, 1, 1) {
+		return fmt.Errorf("You can only specify one instance role ARN with the '--%s' flag", command.InstanceRoleFlag)
```

Comment:
Will it work if only the role name is specified here and not the ARN? If it does, we should modify this error message to say `instance role with..` and get rid of `ARN` part. Unless, you validate that this is an ARN in the `hasCustomRole` method.